src/lib/model/OutboundTransfersModel.js

```diff
@@ -1068,6 +1068,18 @@ class OutboundTransfersModel {
                     await this._save();
                     this._logger.log('State machine in errored state');
                     return;
+
+                case 'aborted':
+                    // stopped in aborted state
+                    await this._save();
+                    this._logger.log('State machine in aborted state');
+                    return this.getResponse();
+
+                default:
+                    // The state is not handled here, throwing an error to avoid an infinite recursion of this function
```

Comment:
This comment indicates that an error should be thrown?

Reply:
I just followed the logic in error case, there its just returning the function. Throwing error might affect the transfer.
I accepted the previous suggestion